components/artwork_metadata_stub/templates/didactics.jade

```diff
@@ -18,6 +18,9 @@ a.artwork-metadata-stub__title.artwork-metadata-stub__line( href= artwork.href )
 if artwork.collecting_institution
   .artwork-metadata-stub__partner.artwork-metadata-stub__line
     = artwork.collecting_institution
+else if artwork.is_in_auction
+  a.artwork-metadata-stub__partner.artwork-metadata-stub__line( href= "/auction#{artwork.partner.href}" )
```

Comment:
It is right, but actually I think the better way to do this is on metaphysics in the partner schema, will close!